src/app/components/home/thumbnail/preview.component.html

```diff
@@ -7,44 +7,32 @@
   @galleryItemAppear
 >
 
-<div
-  class="video-box"
-  [ngStyle]="{ height: imgHeight + 'px' }"
->
-
-  <span *ngIf="showMeta" class="time" @metaAppear>{{ time | lengthPipe }}</span>
-  <span *ngIf="showMeta" class="rez" @metaAppear>{{ rez }}</span>
-
-  <div *ngIf="hoverScrub && hover && noError">
-    <div
-      *ngFor="let item of [0,1,2,3,4,5,6,7,8,9]; let i = index"
-      (mouseover)="showThisOne(i)"
-      [ngStyle]="{'left': 10 * i + '%' }"
-      class="hoverable"
-    >
-    </div>
+  <div
+    #filmstripHolder
+    class="video-box"
+    (mousemove)="mouseIsMoving($event)"
+    [ngStyle]="{
+        height: imgHeight + 'px',
+        'background-image': 'url(' + (hover ? fullFilePath : firstFilePath) + ')',
```

Comment:
I like this, apart from 2 things:

1. Can we make it so that the image doesn't flick back to the first image when you stop hovering?

2. We should sync up the image and video thumbnails (at least the first one) if we're going to do this!

Reply:
1. I made the preview flick back to the original image on purpose as requested by a customer. But since the _settings_ menu is cleaner I can add more option toggles -- this one could be it.
2. I rather like that the video thumbnail image (the `-first.jpg`) is different from any of the ones in the filmstrip. I'm happy to keep them different.

Reply:
1. Yes please! 👍 I like it staying where it is!

2. Hmm, it conflicts with option 1 though, if it doesn't reappear... maybe it isn't an issue.

Reply:
I'll put in a settings button that will allow users to choose between retaining the last-viewed thumbnail or to always come back to the first preview image.

What other fixes would you like to see before we can merge this in? (I know there's more polish and bug-fixing to do before this PR is a fully-operational feature, but I'd rather patch them with subsequent PRs rather than spend many more days on this one) 😃

Reply:
If your happy with bug fixes and changes later - merge away! 🎉 👍